cli-output/src/cli_output.rs

```diff
@@ -1311,6 +1311,84 @@ impl VerboseDisplay for CliStakeState {
     }
 }
 
+fn show_inactive_stake(
+    me: &CliStakeState,
+    f: &mut fmt::Formatter,
+    delegated_stake: u64,
+) -> fmt::Result {
+    if let Some(deactivation_epoch) = me.deactivation_epoch {
+        if me.current_epoch > deactivation_epoch {
+            let deactivating_stake = me.deactivating_stake.or(me.active_stake);
+            if let Some(deactivating_stake) = deactivating_stake {
+                writeln!(
+                    f,
+                    "Inactive Stake: {}",
+                    build_balance_message(
+                        delegated_stake - deactivating_stake,
+                        me.use_lamports_unit,
+                        true
+                    ),
+                )?;
+                writeln!(
+                    f,
+                    "Deactivating Stake: {}",
+                    build_balance_message(deactivating_stake, me.use_lamports_unit, true),
+                )?;
+            }
+        }
+        writeln!(
+            f,
+            "Stake deactivates starting from epoch: {deactivation_epoch}"
+        )?;
+    }
+    if let Some(delegated_vote_account_address) = &me.delegated_vote_account_address {
+        writeln!(
+            f,
+            "Delegated Vote Account Address: {delegated_vote_account_address}"
+        )?;
+    }
+    Ok(())
+}
+
+fn show_active_stake(
+    me: &CliStakeState,
+    f: &mut fmt::Formatter,
+    delegated_stake: u64,
+) -> fmt::Result {
+    if me
+        .deactivation_epoch
+        .map(|d| me.current_epoch <= d)
+        .unwrap_or(true)
+    {
+        let active_stake = me.active_stake.unwrap_or(0);
+        writeln!(
+            f,
+            "Active Stake: {}",
+            build_balance_message(active_stake, me.use_lamports_unit, true),
+        )?;
+        let activating_stake = me.activating_stake.or_else(|| {
+            if me.active_stake.is_none() {
```

Comment:
I think might be simplified:
```rust
me.active_stake.is_none().then_some(delegated_stake)
```

Reply:
I think another PR is best, I just wanted to move the logic as-is to make sure functionality is identical.